lib/Screens/Tasks/completeActivity.dart

```diff
@@ -35,8 +35,13 @@ class _TaskDetails extends State<TaskDetails> {
   Widget build(BuildContext context) {
     final taskObserver = Provider.of<TaskObserver>(context, listen: false);
 
-    responseText = taskObserver.currTaskForDetails!.responseText;
-    print('response text = ' + responseText);
+
+    try {
+      responseText = taskObserver.currTaskForDetails!.responseText;
+      print('response text = ' + responseText);
+    } on Exception catch (e) {
+      print('Exception $e');
```

Comment:
We could have this exception take you back to the Tasks screen instead. This popped up during testing so I surrounded it with a try catch